button-examples/Web/web-button-example.md

```diff
@@ -593,7 +595,7 @@ API and source code:
 
 The following example shows text, outlined and contained button types with Material Theming.
 
-!["Button theming example for <platform> with three buttons - text, outlined and contained - with green/black color theming and cut corners."](assets/button-theming.svg)
+!["Button theming example for <platform> with three buttons - text, outlined and contained - with green/black color theming."](assets/button-theming.svg)
 
 <details>
 <summary><b>Implementing button theming</b></summary>
```

Comment:
@una 

Can you add a code sample and screenshot for the theming example described below the fold?